docs/03-show-order-status.md

```diff
@@ -437,10 +437,11 @@ The `NavigationManager` lets you interact with URIs and navigation state. It has
 
 To use this, update the `PlaceOrder` code so it calls `NavigationManager.NavigateTo`:
 
-```cs
+```csharp
 async Task PlaceOrder()
 {
-    var newOrderId = await HttpClient.PostJsonAsync<int>("orders", order);
+    var response = await HttpClient.PostAsJsonAsync("orders", order);
```

Comment:
Do you think it's worthwhile trying to ask the S.T.Json team to consider a `PostAsJsonAsync<TInput, TOutput>` extension? It got brought up in the API review, but it was dismissed as being cumbersome. But if it's something users may repeatedly do, it's interesting to consider.

Reply:
I don't feel the need to push for it, because it seems like something that can be easily added at some future point if there are a lot of customers asking for it.